### PR TITLE
Improve reporting on robustness/ethical issues [GSK-1765]

### PR DESCRIPTION
**Pull request by mattbit**
- Remove percentage and put x our of N samples changed because we don't know how many samples contain countries
- Add the probabilities next to the prediction

### CODE REVIEW

File: python-client/giskard/scanner/robustness/base_detector.py [hunk 1]
```diff
@@ -134,26 +134,26 @@ def _detect_issues(
...
                     group=self._issue_group,
                     level=issue_level,
                     transformation_fn=transformation_fn,
                     description=desc,
+                    features=[feature],
                     meta={
                         "feature": feature,
                         "domain": f"Feature `{feature}`",
-                        "deviation": f"{round(fail_rate * 100, 2)}% of tested samples changed prediction after perturbation",
+                        "deviation": f"{failed_size} out of {slice_size} tested samples ({round(fail_rate * 100, 2)}%) changed prediction after perturbation",
```
Comment (rabah-khalek) [inline python-client/giskard/scanner/robustness/base_detector.py:154]: we should unify the convention here, I used `{failed_size}/{slice_size}` in the prompt-injection.

Comment (mattbit) [reply python-client/giskard/scanner/robustness/base_detector.py:154]: Let’s keep what is the most clear in your opinion

**Comment by mattbit** [comment]
@rabah-khalek what should we do with this? It’s been a while

File: python-client/giskard/scanner/robustness/base_detector.py (hunk 1 above)
Comment (rabah-khalek) [inline python-client/giskard/scanner/robustness/base_detector.py:154]:
```suggestion
                        "deviation": f"{failed_size}/{slice_size} tested samples ({round(fail_rate * 100, 2)}%) changed prediction after perturbation",
```

**Review by rabah-khalek — APPROVED** [review]

**Comment by rabah-khalek** [comment]
sorry, it's good to go once the conflicts are resolved.

**Comment by mattbit** [comment]
Taking care of this now

**Pull request merged**